tests/test_utility.py

```diff
@@ -9,30 +9,35 @@
     "set_nodes, expected_nodes",
     [
         (
-            "https://firstnode.neurobagel.org/query",
-            ["https://firstnode.neurobagel.org/query/"],
+            "(https://firstnode.neurobagel.org/query, firstnode)",
+            {"https://firstnode.neurobagel.org/query/": "firstnode"},
         ),
         (
-            "https://firstnode.neurobagel.org/query/ https://secondnode.neurobagel.org/query",
-            [
-                "https://firstnode.neurobagel.org/query/",
-                "https://secondnode.neurobagel.org/query/",
-            ],
+            "(https://firstnode.neurobagel.org/query/, firstnode) (https://secondnode.neurobagel.org/query, secondnode)",
+            {
+                "https://firstnode.neurobagel.org/query/": "firstnode",
+                "https://secondnode.neurobagel.org/query/": "secondnode",
+            },
         ),
         (
-            " https://firstnode.neurobagel.org/query/ https://secondnode.neurobagel.org/query/  ",
-            [
-                "https://firstnode.neurobagel.org/query/",
-                "https://secondnode.neurobagel.org/query/",
-            ],
+            "(firstnode.neurobagel.org/query/, firstnode) (https://secondnode.neurobagel.org/query, secondnode)",
+            {
+                "https://secondnode.neurobagel.org/query/": "secondnode",
+            },
```

Comment:
if we don't start with https we want to ignore? I think that should also give a warning in the API log when the .env is read - i.e.: we read some stuff from you that we can't understand, so we'll ignore